lib/data/aws_regions.yaml

```diff
@@ -0,0 +1,73 @@
+---
+ap-northeast-1:
+  :name: ap-northeast-1
+  :hostname: ec2.ap-northeast-1.amazonaws.com
+  :description: Asia Pacific (Tokyo)
+ap-northeast-2:
+  :name: ap-northeast-2
+  :hostname: ec2.ap-northeast-2.amazonaws.com
+  :description: Asia Pacific (Seoul)
+ap-south-1:
+  :name: ap-south-1
+  :hostname: ec2.ap-south-1.amazonaws.com
+  :description: Asia Pacific (Mumbai)
+ap-southeast-1:
+  :name: ap-southeast-1
+  :hostname: ec2.ap-southeast-1.amazonaws.com
+  :description: Asia Pacific (Singapore)
+ap-southeast-2:
+  :name: ap-southeast-2
+  :hostname: ec2.ap-southeast-2.amazonaws.com
+  :description: Asia Pacific (Sydney)
+ca-central-1:
+  :name: ca-central-1
+  :hostname: ec2.ca-central-1.amazonaws.com
+  :description: Canada (Central)
+eu-central-1:
+  :name: eu-central-1
+  :hostname: ec2.eu-central-1.amazonaws.com
+  :description: EU (Frankfurt)
+eu-west-1:
+  :name: eu-west-1
+  :hostname: ec2.eu-west-1.amazonaws.com
+  :description: EU (Ireland)
+eu-west-2:
+  :name: eu-west-2
+  :hostname: ec2.eu-west-2.amazonaws.com
+  :description: EU (London)
+eu-west-3:
+  :name: eu-west-3
+  :hostname: ec2.eu-west-3.amazonaws.com
+  :description: EU (Paris)
+sa-east-1:
+  :name: sa-east-1
+  :hostname: ec2.sa-east-1.amazonaws.com
+  :description: South America (Sao Paulo)
+us-east-1:
+  :name: us-east-1
+  :hostname: ec2.us-east-1.amazonaws.com
+  :description: US East (N. Virginia)
+us-east-2:
+  :name: us-east-2
+  :hostname: ec2.us-east-2.amazonaws.com
+  :description: US East (Ohio)
+us-west-1:
+  :name: us-west-1
+  :hostname: ec2.us-west-1.amazonaws.com
+  :description: US West (N. California)
+us-west-2:
+  :name: us-west-2
+  :hostname: ec2.us-west-2.amazonaws.com
+  :description: US West (Oregon)
+cn-north-1:
+  :name: cn-north-1
+  :hostname: ec2.cn-north-1.amazonaws.com.cn
+  :description: China (Beijing)
+cn-northwest-1:
+  :name: cn-northwest-1
+  :hostname: ec2.cn-northwest-1.amazonaws.com.cn
+  :description: China (Ningxia)
+us-gov-west-1:
+  :name: us-gov-west-1
+  :hostname: ec2.us-gov-west-1.amazonaws.com
+  :description: AWS GovCloud (US)
```

Comment:
I recommend sorting this list by key before saving.  This way, future diffs are easier to read.

Reply:
It possible, but it will make list inconsistent - China's and Gov Cloud's regions will be in the middle of the regular list. Now they're at the bottom, as defined in the AWS gem.
In case of any need to have specific sorting - I suggest to do that manually as needed.